ui/components/privacy-view.jsx

```diff
@@ -0,0 +1,145 @@
+         The cookie, which is unique to your browser, allows the server to “remember” specific information about your visit while you are connected.
+
+        There are two types of cookies — single session (temporary) and multi-session (persistent).
+         Single session cookies last only as long as your Web browser is open.
+         Once you close your browser, the session cookie disappears.
+         Persistent cookies are stored on your device for longer periods.
+         Both types of cookies create an identifier that is unique to your device.
+        The Office of Management and Budget Memorandum M-10-22, Guidance for Online Use of Web Measurement and Customization Technologies, allows Federal agencies to use both session and persistent cookies to improve the delivery of services.
+
+        Session Cookies: We use session cookies for technical purposes, such as to allow better navigation through our site.
+         These session cookies are set by GSA or a contractor acting on GSA’s behalf and let the server hosting the OMB Policy Library know that you are continuing a visit to our site.
+         Our use of session cookies qualifies as “Usage Tier 1—Single Session,” as defined in the OMB M-10-22 guidance.
+        Persistent Cookies: We use persistent cookies to understand the differences between new and returning visitors to the OMB Policy Library.
+        Persistent cookies remain on your device between visits to our site until they expire or are removed by the user.
+         As discussed below under “Google Analytics,” our use of persistent cookies qualifies as “Usage Tier 2—Multi-session without personally identifiable information,” as defined in the OMB M-10-22 guidance.
+         The policy states, “This tier encompasses any use of multi-session Web measurement and customization technologies when no [personally identifiable information] is collected.” We do not use persistent cookies to collect personally identifiable information.
+        If you do not want to accept cookies, you can edit your browser’s options to stop accepting persistent cookies or to prompt you before accepting a cookie from the websites you visit.
+         Here are instructions for how you can disable cookies and/or Google Demographic and Interests reports.
+         {/*eslint-enable */}
+      </p>
+
+
+      <h4>Google Analytics</h4>
+      <p className="max-width-3">
+        {/*eslint-disable */}
+        Fpc.gov participates in the U.S.
+         Digital Analytics Program, (DAP) which utilizes a unified Google Analytics account for Federal Government agencies.
+         This program helps Federal agencies understand how people find, access, and use government services online.
+
+        The DAP is a hosted shared service provided by the General Services Administration’s (GSA’s) Office of Citizen Services and Innovative Technologies, and the protocol and information collected are the same for all websites participating in the DAP.
```

Comment:
replace `Office of Citizen Services and Innovative Technologies` with `Technology Transformation Service` (just in case there's another re-org)